README.rst

```diff
@@ -268,6 +268,54 @@ There is another entry point for Android Studio - ``src/examples/facefilter/andr
 It should be used only for testing or as a template for starting your own project
 based on Drishti.
 
+Android Studio workarounds
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next factors add an instability to the Android Studio build:
+
+* Android Studio 3.2 is **beta**
+* Using custom CMake 3.7+ in Android Studio is `a preview feature <https://developer.android.com/studio/projects/add-native-code#vanilla_cmake>`__
+* Some issues are hard to track or confirm, some issues `like this one <https://issuetracker.google.com/issues/75268076>`__ is already reported but still **not fixed**
+
+From experience the weakest part in build is a communication between Gradle and
+CMake, to minimize it the next trick can be used:
+
+* Open top-level ``CMakeLists.txt`` file
+* Find ``if(DRISHTI_DEBUG_STOP)`` `condition <https://github.com/elucideye/drishti/blob/d8b91e26eb1a1f62412bd2d56d1a229d646b6864/CMakeLists.txt#L102-L107>`__
+* Substitute ``if(DRISHTI_DEBUG_STOP)`` with ``if(TRUE)``
+* Run Gradle build:
+
+.. code-block:: none
+
+  [drishti]> cd android-studio
+  [drishti/android-studio]> ./gradlew assembleDebug
+
+If you're running it a first time there will be a high chance to hit this
+Gradle issue:
+
+.. code-block:: none
+
+  * What went wrong:
+  Execution failed for task '...'.
+  > Conversion = c, Flags =
+
+In this case just wait for few seconds and run Gradle again:
+
+.. code-block:: none
+
+  [drishti/android-studio]> ./gradlew assembleDebug
+
+* Revert ``CMakeLists.txt`` file, i.e. substitute ``if(TRUE)`` with ``if(DRISHTI_DEBUG_STOP)``.
+
+* Run CMake build without Gradle:
+
+.. code-block:: none
+
+  [drishti/android-studio]> cmake --build ../src/examples/facefilter/android-studio/app/.externalNativeBuild/cmake/debug/arm64-v8a
+
+Once CMake build is ready you can use ``./gradlew assembleDebug`` or open
```

Comment:
Once >>the<< CMake build is ready>>,<< you can use

Reply:
While this is less than ideal, the builds have been manageable in practice, and the **beta** issues should be resolved over time.